### ISSUE TITLE
Site stops working when selecting normalization option 'none'

### ISSUE THREAD

**Issue by jonco3** [mozilla/glam#2774]
STR:
 - Nativgate to https://glam.telemetry.mozilla.org
 - select telemetry probe 'gc_minor_us'
 - select channel 'release'
 - select normalization 'none'

Result: 

The site becomes unresponsive. The console reports the following exception was thrown:
```
Uncaught (in promise) TypeError: n[(n.length - 1)][Gn(...)].map is not a function
    l QuantileExplorerView.svelte:77
    update QuantileExplorerView.svelte:89
    xt index.mjs:1343
    _t index.mjs:1307
```

**Comment by edugfilho** [comment]
Thanks for reporting! 
The probe doesn't have non-normalized data and that's breaking the page. Still investigating why there's no normalized data and why that's not being handled properly.

**Comment by edugfilho** [comment]
[Probe Dictionary](https://probes.telemetry.mozilla.org/?view=detail&probeId=histogram/GC_MINOR_US) says this probe's population is `prerelease`, and the `Expiry` section doesn't mention `release`. @jonco3 would that mean this probe is not supposed to have data for release?
In any case, GLAM must handle it appropriately. Still investigating

**Comment by jonco3** [comment]
I guess it's 'prerelease' because this probe doesn't have {"releaseChannelCollection": "opt-out"} in histograms.json and so it doesn't get release channel data by default. I didn't realise that this was the case. 

But there is obviously some release channel data there, because it's displaying the normalized version.

**Comment by edugfilho** [comment]
GLAM breaks when `normalization: none` is selected because there's no non-normalized data for that probe (behavior which needs to be fixed), and that is because GLAM only [started aggregating non-normalized data](https://github.com/mozilla/bigquery-etl/pull/3873/files) after it stopped collecting data for that probe. Notice that we did not backfill non-normalized data because that was prohibitively expensive, time and money-wise.

Looking at the raw data I learned GLAM was never supposed to have collected `release` data for the probe in question at all, due to the low client counts. The reason why there are two builds there is because those builds were aggregated before a [better client count filtering was put in place for release](github.com/mozilla/glam/issues/1575).

For info, [the probe seems to be still collecting data for a recent build](https://sql.telemetry.mozilla.org/queries/98719/source), in case you want to dig further.


In summary: 
- GLAM stopped collecting `release` data for that probe in Oct '21, and the `release` data you see on GLAM now is what was collected and aggregated prior to that date. 
- GLAM has no non-normalized `release` data for that probe because when we introduced non-normalized aggregations to GLAM we did not run a backfill
- [We'll work on handling `normalization: none` for probes without non-normalized data](https://github.com/mozilla/glam/issues/2777)
